mlabs/efficient-nft-pab/Main.hs

```diff
@@ -176,6 +177,9 @@ main = do
   Hask.putStr "Unapplied sgNFT minting policy: "
   ByteString.putStrLn $ JSON.encode Token.policyDataScript
 
+  Hask.putStr "Unapplied locking validator: "
+  ByteString.putStrLn $ JSON.encode Lock.lockScriptUntyped
```

Comment:
I'm not sure if `JSON.encode` gives the same result as `Text.decodeUtf8 . Base16.encode $ serialise`. I used the latter for the locking script saved in CTL right now

Reply:
Looked into it, they both calldown to the Serialise typeclass, but unsure about the encoding, I'll switch to this method for all of them